lib/describe-metadata-result.json

```diff
@@ -685,7 +685,13 @@
 		"metaFile": false,
 		"suffix": "lightningExperienceThemes",
 		"xmlName": "LightningExperienceTheme"
-	}],
+	}, {
+		"directoryName": "notificationtypes",
+		"inFolder": false,
+		"metaFile": false,
+		"suffix": "notificationtypes",
```

Comment:
Shouldn't this be `"suffix": "notiftype"`?
`suffix` is the file extension and looking at any of my sfdx orgs in `.sfdx/USERNAME/metadataTypeInfos.json` i see the following:

```
        "CustomNotificationType": {
            "metadataName": "CustomNotificationType",
            "isAddressable": true,
            "isSourceTracked": true,
            "ext": "notiftype",
            "hasContent": false,
            "defaultDirectory": "notificationtypes",
            "nameForMsgs": "Custom Notification Type",
            "nameForMsgsPlural": "Custom Notification Types",
            "contentIsBinary": false,
            "hasStandardMembers": false,
            "deleteSupported": true,
            "decompositionConfig": {
                "strategy": "nonDecomposed",
                "workspaceStrategy": "nonDecomposed",
                "commitStrategy": "fineGrainTracking",
                "metadataName": "CustomNotificationType",
                "useSparseComposition": false,
                "decompositions": [],
                "contentStrategy": "nonDecomposedContent"
            },
            "hasVirtualSubtypes": false
        },
```